packages/components/src/summary-button/style.scss

```diff
@@ -77,6 +77,7 @@
 
 		.summary-button-title {
 			@include body-medium;
+			line-height: $font-line-height-medium;
```

Comment:
Presuming this variable already exists, and I guess in being SCSS it's more private, but my only feedback is that this is a bit of a curious variable name from a design system perspective.

In fact maybe the entire line should be _inside_ the `body-medium` mixin instead. Is there any reason it isn't?

Reply:
It's a recent mix-in, part of the design system work that @jameskoster was doing, and most likely something we can change.